### PR TITLE
#7 Task. Sergey Starushkin

### PR DESCRIPTION
**Pull request by Sergey-Starushkin**

### CODE REVIEW

File: task/07-yield-tasks.js [hunk 1]
```diff
@@ -126,7 +156,22 @@ function* breadthTraversalTree(root) {
...
+    const first = source1();
+    const second = source2();
+
+    while (true) {
+        let a = first.next().value;
+        let b = second.next().value;
+
+        if (a == undefined)
+            yield b;
+        else if (b == undefined)
```
Comment (vramaniuk) [inline task/07-yield-tasks.js:168]: "operator === is always preferable
sometimes ''==' may cause an error
(check at console :
'0' == 0
and '0' === 0)"

**Review by vramaniuk — COMMENTED** [review]
Please, privide the direct link to your task success travis build

I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by Sergey-Starushkin** [comment]
https://travis-ci.org/github/Sergey-Starushkin/js-assignments/builds/722849229

**Comment by vramaniuk** [comment]
Please, see the notes for tasks1-4. (dirty PR)
and don't forget implement task 7 note (about strict equality)

**Comment by vramaniuk** [comment]
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by vramaniuk** [comment]
Selection_022
These files shoudn't be changed in this PR.
Please,revert commits, that contain changes in these files
Check if it's ok in this section: 
https://github.com/AisBrestEDU/js-assignments/pull/813/files

I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Pull request closed without merging**